WoWsShipBuilder.Web/Pages/Index.razor

```diff
@@ -91,17 +101,45 @@
 
     private void OnLoadBuild()
     {
-        try
+        if (buildString.Contains("share.wowssb.com"))
```

Comment:
Again, do not hardcode urls or hostnames unless their structure is defined in code in the same project (like the name of a specific page)